src/bridge/AdminRoomHandler.ts

```diff
@@ -74,6 +74,11 @@ const COMMANDS: {[command: string]: Command|Heading} = {
         summary: "Leave all bridged channels, on all networks, and remove your " +
                 "connections to all networks.",
     },
+    "!active": {
+        example: "!active",
+        summary: "Informs the bridge that you do not wish to be idlekicked. This will reset your idleness timer " +
+                 "but you may need to run this again."
```

Comment:
This reads to me a little like "try typing it in a few times just in case". We may want to clarify when's a good time for the "again".